### PR TITLE
feat: :sparkles: Wallet functionality for ICRC 1 tokens

### PR DESCRIPTION
**Pull request by Jair-MoralesN**
Wallet to manage their digital assets and perform transactions with major ICRC1 cryptocurrencies

User-friendly interface for ICRC 1 tokens

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,107 @@
...
+- [License](#license)
+
+## Getting Started
+
+To get started with HPL Wallet, follow these steps:
+
+1. Clone the repository to your local machine:
+
+   ```
+   git clone https://github.com/SDS-Timo/wallet.git
```
Comment (timohanke) [inline README.md]: git clone git@github.com:research-ag/wallet.git

File: README.md [hunk 2]
```diff
@@ -1 +1,107 @@
...
+   cd wallet
+   ```
+
+3. Install the project dependencies:
+
+   ```
+   npm install
+   ```
+
+4. Create .env file in the root of the proyect:
```
Comment (timohanke) [inline README.md]: proyect -> project

File: README.md [hunk 3]
```diff
@@ -1 +1,107 @@
...
+
+   ```
+   NODE_ENV=PRODUCTION
+   # Ledger canister for ICP
+   VITE_ICP_LEDGER_CANISTER_ID="ryjl3-tyaaa-aaaaa-aaaba-cai"
+   # Transacction history API for ICP
```
Comment (timohanke) [inline README.md]: transacction -> transaction

File: frontend/assets/svg/files/decred-icon.svg
Comment (timohanke) [inline frontend/assets/svg/files/decred-icon.svg:1]: delete the file

File: frontend/assets/img/fnid-logo.png
Comment (timohanke) [inline frontend/assets/img/fnid-logo.png:1]: should this file be named nfid-logo.png?

File: frontend/assets/svg/files/binance-icon.svg
Comment (timohanke) [inline frontend/assets/svg/files/binance-icon.svg:1]: delete this file

File: frontend/assets/svg/files/litecoin-icon.svg
Comment (timohanke) [inline frontend/assets/svg/files/litecoin-icon.svg:1]: delete this file

File: frontend/assets/svg/files/neo-icon.svg
Comment (timohanke) [inline frontend/assets/svg/files/neo-icon.svg:1]: delete this file

File: frontend/const.ts [hunk 4]
```diff
@@ -0,0 +1,157 @@
+// svg
+import IcIcon from "@/assets/img/icp.png";
+import EthereumIcon from "@/assets/svg/files/ethereum-icon.svg";
+import NeoIcon from "@/assets/svg/files/neo-icon.svg";
+import DecredIcon from "@/assets/svg/files/decred-icon.svg";
+import BinanceIcon from "@/assets/svg/files/binance-icon.svg";
+import LitecoinIcon from "@/assets/svg/files/litecoin-icon.svg";
```
Comment (timohanke) [inline frontend/const.ts]: Remove these lines for irrelevant altcoins

File: frontend/const.ts [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+export const DeleteContactTypeEnum = z.enum(["CONTACT", "ASSET", "SUB"]);
+export type DeleteContactTypeEnum = z.infer<typeof DeleteContactTypeEnum>;
+
+export const TimerActionTypeEnum = z.enum(["TRANSACTIONS", "ASSETS"]);
+export type TimerActionType = z.infer<typeof TimerActionTypeEnum>;
+
+//
+export const ICRC1systemAssets: Array<Token> = [
+  {
+    id_number: 9999,
```
Comment (timohanke) [inline frontend/const.ts]: Is this ok to have multiple with id_number 9999? Can it cause any problems?

File: frontend/const.ts [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+};
+
+export const symbolIconDict: { [key: string]: string } = {
+  BTC: BitcoinIcon,
+  ICP: IcIcon,
+  ETH: EthereumIcon,
+  NEO: NeoIcon,
+  DCR: DecredIcon,
+  BNB: BinanceIcon,
+  LTC: LitecoinIcon,
```
Comment (timohanke) [inline frontend/const.ts]: remove these lines

File: frontend/const.ts [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+    index: "6dfr2-giaaa-aaaaq-aaawq-cai",
+  },
+];
+
+export const DIP20systemAssets: Array<Token> = [];
+export const EXTsystemAssets: Array<Token> = [];
+export const systemAssets: { [key: string]: Array<Token> } = {
+  "ICRC-1": ICRC1systemAssets,
+  DIP20: DIP20systemAssets,
+  EXT: EXTsystemAssets,
```
Comment (timohanke) [inline frontend/const.ts]: Do we use dip20 and ext for anything? Do we need these lines?

**Review by timohanke — COMMENTED** [review]
Left some comments and questions

File: README.md [hunk 8]
```diff
@@ -1 +1,107 @@
...
+Once you have installed the project and started the development server, you can access the HPL Wallet application in your web browser. Here are some basic usage instructions:
+
+- Create a new wallet and secure your wallet information.
+- Add your cryptocurrencies and manage your digital assets.
+- Send and receive cryptocurrencies to and from other users.
+- Explore real-time price charts and market data.
+
+## License
+
+This project is licensed under the GPL3 License - see the [COPYING](COPYING) file for details.
```
Comment (timohanke) [inline README.md]: LICENSE file, not COPYING

File: frontend/const.ts (hunk 5 above)
Comment (Jair-MoralesN) [reply frontend/const.ts]: Just a initial number, then when a token is added a new id will be asigned. However I'll change a different id for each.

**Pull request merged**